### PR TITLE
Fix upgrade script v02_02_000.py

### PR DESCRIPTION
**Pull request by Bugerman58**
## Description of the issue/feature this PR addresses

Upgrade script updating worksheet layouts for exists worksheet but not for bika_setup

## Current behavior before PR

Not updating for default worksheet layout in the bika_setup

## Desired behavior after PR is merged

Update worksheet layouts for bika_setup too

--
I confirm I have tested this PR thoroughly and coded it according to [PEP8][1]
and [Plone's Python styleguide][2] standards.

[1]: https://www.python.org/dev/peps/pep-0008
[2]: https://docs.plone.org/develop/styleguide/python.html

### CODE REVIEW

File: src/bika/lims/browser/worksheet/tools.py [hunk 1]
```diff
@@ -90,6 +90,6 @@ def getWorksheetLayouts():
     """
     layouts = []
     for name, layout_utility in getUtilitiesFor(IWorksheetLayouts):
-        map(lambda entry: layouts.append(entry), layout_utility.getLayouts())
+	    [layouts.append(layout) for layout in layout_utility.getLayouts()]
```
Comment (ramonski) [inline src/bika/lims/browser/worksheet/tools.py]: Can you fix the indendation to 4 spaces here please

File: src/senaite/core/upgrade/v02_02_000.py [hunk 2]
```diff
@@ -214,5 +214,12 @@ def migrate_worksheet_layouts(portal):
         if layout:
             # set the new layout
             obj.setResultsLayout(layout)
+    
+    logger.info("Update default worksheet layout for BikaSetup ...")
+
+    bika_setup = api.get_bika_setup()
```
Comment (ramonski) [inline src/senaite/core/upgrade/v02_02_000.py]: just a minor thing:
we're fading out the `bika` namespace, so better use `setup = api.get_setup()`

**Review by ramonski — CHANGES_REQUESTED** [review]

File: src/senaite/core/upgrade/v02_02_000.py (hunk 2 above)
Comment (Bugerman58) [reply src/senaite/core/upgrade/v02_02_000.py]: done

File: src/bika/lims/browser/worksheet/tools.py (hunk 1 above)
Comment (Bugerman58) [reply src/bika/lims/browser/worksheet/tools.py]: sorry.
done

**Review by ramonski — APPROVED** [review]

**Pull request merged**